### DISCUSSION TITLE
Notice viewer goal

### DISCUSSION THREAD

**Discussion by DKamerbeek** [OP-TED/eForms-SDK#81] [Q&A] [answered]
I'm having doubts about the readyness of the notice viewer application to use in production when the SDK is stable.

https://docs.ted.europa.eu/eforms/1.0.0/notice-viewer/index.html highlights the following:
![image](https://user-images.githubusercontent.com/32478527/183414833-3ff45071-5938-4c5c-bed2-8335c429e95d.png)

Should i interpret this as: 

- For now the notice viewer application is unstable, but this should be fixed in the near future. Consumers can use this app for generating html in production.

or

- The notice viewer application is only for demonstration usage, we don't have the intention to make this application production ready so that a client can use this viewer in their production code.

**Comment by rousso** [comment, marked as answer]
Hi @DKamerbeek,

The notice viewer is a sample application. There is no need to have any doubts about it. 😉 

We thought that it was clear enough that it is a **sample** by the title we chose for [the specific docs page that you refer to ](https://docs.ted.europa.eu/eforms/1.0.0/notice-viewer/index.html) (which reads _"Notice Viewer Sample Application"_) as well as the [README.md](https://github.com/OP-TED/eforms-notice-viewer/blob/develop/README.md) file inside the [eforms-notice-viewer git repository](https://github.com/OP-TED/eforms-notice-viewer) which starts with _"This is a sample command line application which demonstrates how you can use the eForms SDK in a metadata driven application that visualises eForms notices."_.  I am happily however reaffirming this to you and I want to leave no doubt to anyone that this is just a sample application.

Having made that clear, it does not mean that we do not care about how it is perceived by developers or how it can be made more useful to developers. I will be happy therefore to answer any specific questions you may have about the things that seem wrong about the notice viewer to you, and to consider any specific suggestions that you may have on how we can improve it.

Thank you for your message,
Yannis
